census.go

```diff
@@ -489,6 +489,12 @@ func (v *vocdoniHandler) checkTokens(tokens []*CensusToken) error {
 		if !ok {
 			return fmt.Errorf("invalid blockchain for token %s provided", token.Address)
 		}
+		// check max holders
+		if holders, err := v.airstack.NumHoldersByTokenAnkrAPI(token.Address, token.Blockchain); err != nil {
```

Comment:
Please do not make the ANKR API token mandatory. If not defined, the code might just ignore this check.